lib/signalfx/tracing/instrumentation/pg.rb

```diff
@@ -0,0 +1,41 @@
+require_relative '../sfx_logger'
+
+module SignalFx
+  module Tracing
+    module Instrumenter
+      module PG
+
+        @logger = SFxLogger.new
+        @logger.log.info('SFx pg-instrumentation') { "Initializing instrumentation ..." }
```

Comment:
Statements that don't require user response are often debug level.  Users generally prefer minimal logging.